Dockerfile

```
@@ -30,15 +30,21 @@ WORKDIR /app
 COPY package.json package-lock.json ./
 RUN npm ci
 
-# Install Playwright's Chromium + create symlink for browser.ts compatibility
+# Teams (and any non-Meet) use official Playwright Chromium via
+# /usr/bin/google-chrome; Meet uses CloakBrowser's own stealth Chromium fork
+# (baked below). Both are installed so browser.ts can pick per provider.
 RUN npx playwright install chromium && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)